### ISSUE TITLE
Gradle + Ant wsimport task + wsdllocation + fluent API fail with: "Argument "-wsdllocation" not supported by "fluent-builder" plugin"

### ISSUE THREAD

**Issue by gavenkoa** [mklemm/jaxb2-rich-contract-plugin#47]
I am trying to integrate plugin into Gradle build to get fluid API for XSD mapped classes.

`wsimport` is executed via Ant task in Gradle:

* define classpath for tools, like:

```
dependencies {
    jaxWsTools 'org.glassfish.metro:webservices-tools:2.4.3'
}
```
or:

```
dependencies {
    jaxWsTools 'com.sun.xml.ws:jaxws-tools:2.3.2'
}
```

* define ant task:

```
        ant.taskdef(name: "wsimport", classname: "com.sun.tools.ws.ant.WsImport", classpath: configurations.jaxWsTools.asPath)
```

* call task:

```
ant.wsimport(...)
```

I successfully generated fuelt API with `jaxb2-rich-contract-plugin` with:

```
ant.wsimport(
    keep: "true",
    xdebug: "true",
    verbose: "true",
    extension: "true"
) {
  xjcarg(value: "-Xfluent-builder")
}
```

Important part from `gradle test -i` build log:

```
[ant:wsimport] JAX-WS RI 2.3.2
[ant:wsimport] command line: wsimportC:\opt\Java\jdk1.8.0_202\jre\bin\java.exe -extension -keep -s C:\home\devel\binding\src\main\java -verbose -Xdebug -Xnocompile -target 2.1 C:\home\devel\binding\src\main\resources\System.wsdl -B-Xfluent-builder
```

I have a problem *Argument "-wsdllocation" not supported by "fluent-builder" plugin.* when I add `wsdlLocation` parameter to gradle action:

```
ant.wsimport(
    keep: "true",
    xdebug: "true",
    verbose: "true",
    wsdlLocation: "/System.wsdl", // <== HERE
    extension: "true"
) {
  xjcarg(value: "-Xfluent-builder")
}
```

Important part of build log for `gradle test -d`:

```
[ant:wsimport] JAX-WS RI 2.3.2
[ant:wsimport] command line: wsimportC:\opt\Java\jdk1.8.0_202\jre\bin\java.exe -extension -keep -s C:\home\devel\binding\src\main\java -verbose -Xdebug -Xnocompile -target 2.1 C:\home\devel\binding\src\main\resources\System.wsdl -B-Xfluent-builder -wsdllocation /System.wsdl
[ant:wsimport] fluent-builder: Argument "-wsdllocation" not supported by "fluent-builder" plugin.
```

While this is not issue with plugin itself I wonder if someone know solution for this issue...

Without `wsdllocation` it is not convenient to instantiate services. Instead no-arg constructor I need to use `SystemService(URL wsdlLocation, QName serviceName)` because default `wsdlLocation = "file:/C:/home/devel/...`. When relative path is used like `/System.wsdl`  wsimport generates classpath related search code:

```
 SYSTEMSERVICE_WSDL_LOCATION = com.travelport.service.system_v32_0.SystemService.class.getResource("/System.wsdl");
```
when with full local path:

```
 SYSTEMSERVICE_WSDL_LOCATION = new URL("file:/C:/home/devel/sa-travelport/binding/src/main/resources/System.wsdl");
```

**Comment by gavenkoa** [comment]
It is actually issue in https://github.com/mklemm/jaxb-plugin-lib that prevents from passing arguments beyond `-Xfluent-builder` option for `wsimport` util:

`com.kscs.util.plugins.xjc.base.AbstractPlugin`:

	private boolean isEndOfPluginArgs(final String arg) {
		return AbstractPlugin.XJC_STANDARD_ARGS.contains(arg) || arg.startsWith("-X") || arg.startsWith("-B-X");
	}

`metro-jax-ws/jaxws-ri/tools/wscompile/src/main/java/com/sun/tools/ws/wscompile/WsimportOptions.java` passes `wsimport` options literary to `com.sun.tools.xjc.Options.parseArgument`:
        } else if (args[i].startsWith("-B")) {
            // JAXB option pass through.
            String[] subCmd = new String[args.length-i];
            System.arraycopy(args,i,subCmd,0,subCmd.length);
            subCmd[0] = subCmd[0].substring(2); // trim off the first "-B"

            com.sun.tools.xjc.Options jaxbOptions = schemaCompiler.getOptions();
            try {
                int r = jaxbOptions.parseArgument(subCmd, 0);

and  plugin `parseArgument()` eats everything:

	private int parseOptions(final String[] args, final int startIndex) throws BadCommandLineException {
		int count = 0;
		for (int i = startIndex; i < args.length && !isEndOfPluginArgs(args[i]); i++) {

There is workaround - to add another plugin to the list after fluent one (to trigger `-B-X`):

            xjcarg(value: "-Xfluent-builder")
            xjcarg(value: "-Xlocator")

or to pass safe common to both utilities option (to trigger `AbstractPlugin.XJC_STANDARD_ARGS.contains`):

            xjcarg(value: "-Xfluent-builder")
            arg(value: "-verbose")

Error message corresponds to plugin's support library:

    jaxb-plugin-lib/src/main/resources/com/kscs/util/plugins/xjc/base/AbstractPlugin.properties
    27:exception.unrecognizedArgument={0}: Argument "{1}" not supported by "{0}" plugin.

**Comment by mklemm** [comment]
Fixed in 4.0.0, see README
